File: algorithms/algorithms-cplusplus/leetcode/Question_0389_Find_the_Difference.cc
#include <string>
#include <vector>

using namespace std;

class Solution {
 public:
  char findTheDifference(string s, string t) {
    int count = t[t.size() - 1];
    for (int i = 0; i < s.size(); i++) {
      count -= s[i];
      count += t[i];
    }
    return (char)count;
  }
};

File: algorithms/algorithms-java/src/main/java/com/lanceolata/coding/algorithms/java/leetcode/Question_0389_Find_the_Difference.java
package com.lanceolata.coding.algorithms.java.leetcode;

public class Question_0389_Find_the_Difference {
    public char findTheDifference(String s, String t) {
        int count = t.charAt(t.length() - 1);
        for (int i = 0; i < s.length(); i++) {
            count -= s.charAt(i);
            count += t.charAt(i);
        }
        return (char)count;
    }
}
